lib/experimental/Navigation/Header/Header/index.tsx

```diff
@@ -0,0 +1,86 @@
+import { Button } from "@/components/Actions/Button"
+import { IconType } from "@/components/Utilities/Icon"
+import AlignTextJustify from "@/icons/AlignTextJustify"
+import { cn } from "@/lib/utils"
+import Breadcrumbs, { type BreadcrumbItemType } from "../Breadcrumbs"
+
+import {
+  ModuleAvatar,
+  type IconType as ModuleAvatarIconType,
+} from "@/experimental/Information/ModuleAvatar"
+
+type HeaderProps = {
+  module: {
+    name: string
+    href: string
+    icon: ModuleAvatarIconType
+  }
+  tree?: BreadcrumbItemType[]
+  actions?: {
+    label: string
+    icon: IconType
+    onClick: () => void
+  }[]
+  menu: {
+    show: boolean
+    onClick: () => void
+  }
+}
+
+export default function Header({
+  module,
+  tree = [],
+  actions = [],
+  menu,
+}: HeaderProps) {
+  const breadcrumbsTree: BreadcrumbItemType[] = [
+    { label: module.name, href: module.href, icon: module.icon },
+    ...tree,
+  ]
+
+  const isFirstLevel = !tree.length
```

Comment:
Careful with this - in JavaScript, `0` is `falsey`. Which means that `!0 == true`. I don't think it matters in this case, but it reads weird. Maybe `tree.length > 0` would be better?

Reply:
Ah JavaScript is weird, haha. Changed the check.